src/OrchardCore.Modules/OrchardCore.OpenId/Validators/OpenIdUrlValidator.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace OrchardCore.OpenId.Validators
+{
+    public static class OpenIdUrlValidator
+    {
+        public static IEnumerable<ValidationResult> ValidateUrls(string memberName, string member)
+        {
+            if (member != null)
+            {
+                foreach (var url in member.Split(new[] { ' ', ',' }, StringSplitOptions.RemoveEmptyEntries))
+                {
+                    if (!Uri.IsWellFormedUriString(url, UriKind.Absolute) || !Uri.TryCreate(url, UriKind.Absolute, out var createdUri))
```

Comment:
Not blocking, but FYI, `Uri.IsWellFormedUriString()` creates an `Uri` internally to validate it so you end up creating the `Uri` twice: https://referencesource.microsoft.com/#System/net/System/UriExt.cs,427

Some like `!Uri.TryCreate(url, UriKind.Absolute, out Uri uri) || !uri.IsWellFormedOriginalString()` would be more efficient (tho' I agree it doesn't matter much here).